lib/clerk/rack_middleware_v2.rb

```diff
@@ -163,11 +163,14 @@ def call(env)
       end
 
       token = verify_token(cookie_token)
+      return signed_out(env) unless token
 
-      if token && token["iat"] && client_uat && Integer(client_uat) <= token["iat"]
+      if token["iat"] && client_uat && Integer(client_uat) <= token["iat"]
```

Comment:
:question: Why did we remove the check for `token` presence? Is it guaranteed that it will never be `nil`?

Reply:
because i have added as guard the case of token being falsy (check updated line 166 above)

Reply:
I need :eyeglasses:. Missed that, thanks.